docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+services:
+  postgres:
+    image: postgres:latest
+    volumes:
+      - db_data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_DB=postgres
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    ports:
+      - "5432:5432"
+    networks:
+      - default
+    deploy:
+      resources:
+        limits:
+          cpus: "2"
+          memory: 2G
+      restart_policy:
+        condition: on-failure
```

Comment:
These constraints are not needed. In my shared compose file, I had this to test Engine running on such a configuration.

Reply:
what should I do with these for local set up? Should I remove?